src/bin/gtfs2ntfs.rs

```diff
@@ -65,8 +66,10 @@ fn run() -> Result<()> {
         transit_model::gtfs::read_from_url(&url, opt.config_path, opt.prefix)?
     } else if opt.input.is_file() {
         transit_model::gtfs::read_from_zip(opt.input, opt.config_path, opt.prefix)?
-    } else {
+    } else if opt.input.is_dir() {
         transit_model::gtfs::read_from_path(opt.input, opt.config_path, opt.prefix)?
+    } else {
+        bail!("Invalid input data: must be an existing directory or a ZIP archive");
```

Comment:
or an url

Reply:
Fix for `gtfs2ntfs` and `kv12ntfs` in 0548c88614d80ab3711833a25529ac9fee089d97.